packages/page-staking/src/Performance/CurrentList.tsx

```diff
@@ -0,0 +1,198 @@
+// Copyright 2017-2022 @polkadot/app-staking authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { SortedTargets, ValidatorInfo } from '../types';
+
+import React, { useMemo, useRef, useState } from 'react';
+
+import { DeriveSessionProgress } from '@polkadot/api-derive/types';
+import { Table } from '@polkadot/react-components';
+import { useApi, useCall, useLoadingDelay } from '@polkadot/react-hooks';
+import { EraIndex } from '@polkadot/types/interfaces';
+import { Option, Struct, u32 } from '@polkadot/types-codec';
+
+import Filtering from '../Filtering';
+import { useTranslation } from '../translate';
+import Address from './Address';
+
+interface Props {
+  className?: string;
+  favorites: string[];
+  targets: SortedTargets;
+  toggleFavorite: (address: string) => void;
+  session?: number;
+  currentSessionCommittee: string[];
+  sessionValidatorBlockCountLookup: Record<string, number>;
+}
+
+type AccountExtend = [string, boolean];
+
+interface Filtered {
+  validators?: AccountExtend[];
+}
+
+function sortAccountByFavourites (accounts: string[] = [], favorites: string[]): AccountExtend[] {
+  return accounts
+    .map((accountId): AccountExtend => [
+      accountId,
+      favorites.includes(accountId)
+    ])
+    .sort(([, isFavA]: AccountExtend, [, isFavB]: AccountExtend): number => {
+      return isFavA === isFavB ? 0 : (isFavA ? -1 : 1);
+    });
+}
+
+function getFiltered (targets: SortedTargets | undefined, favorites: string[], currentSessionCommittee: string[]): Filtered {
```

Comment:
```suggestion
function getFiltered(targets: SortedTargets | undefined, favorites: string[], currentSessionCommittee: string[]): Filtered {
```

Reply:
I think `yarn lint` does it, other functions touched in the PR also have space